numpy/ma/core.py

```diff
@@ -5868,6 +5868,7 @@ def __deepcopy__(self, memo=None):
         memo[id(self)] = copied
         for (k, v) in self.__dict__.items():
             copied.__dict__[k] = deepcopy(v, memo)
+        copied._sharedmask = False
```

Comment:
Same for this line, again, rather unreliable stuff since a copy may be made at some point, could remove it.